### ISSUE TITLE
Password Field populating / validation

### ISSUE THREAD

**Issue by Zauberfisch** [swipestripe/silverstripe-swipestripe#39]
when the OrderForm is submitted and returns with a validation error, the Password Field is populated with the submitted password.

The problem is that the field consist out of 2 inputs that check that you have entered the correct password twice.
now lets assume you enter the following:
1. somepass
2. someotherpass

after submitting, it will display an error like "Passwords have to match", but populates the field with:
1. somepass
2. somepass

so I could just resubmit the form and now it would work.
this however is a big problem, because if I misspelled my first password and accidentally do that 2nd submit, my account would be created with a wrong password.

In my opinion the password field should either not be populated at all, or only if the passwords matched.

**Comment by frankmullenger** [comment]
@Zauberfisch agreed, populate only if the passwords match would be ideal I think.
